book/src/coding-conventions.md

```diff
@@ -70,3 +70,17 @@ TODO: how to name error types/error enum variants (`CannotDoSth` vs `DoingSth`)?
 ## Dependencies
 
 If the same dependency is used in multiples crates within the workspace, that dependency SHOULD be specified in the *workspace*'s `Cargo.toml` file and workspace crates should import them from there.
+
+## Adding a new workspace crate, exposed by `riot-rs`
+
+To add a new workspace crate re-exported by `riot-rs`, follow these steps:
+
+1. Create the new crate's directory in `src/`.
+1. Run `cargo init --lib` in that directory.
+1. In the new crate's `Cargo.toml`, move the automatically added  `[lints]` section above the `[dependencies]` section.
+1. Add a `#![deny(clippy::pedantic)]` lint to the new crate's root module, along with other relevant lints (e.g., `#![deny(missing_docs)]`); some lints are already inherited from the workspace and do not need to be added to the new crate.
```

Comment:
Why require the `#![deny(clippy::pedantic)]`? We only have it in `riot-rs-benchmarks` and `riot-rs-macros` right now. 
Generally, I don't really think this whole line is necessary; it's nothing RIOT-rs specific.

Reply:
> not having the `pedantic` lint is not the project's policy

Is there a related discussion somewhere, where I can read up upon the reasoning?